src/backend/marsha/core/migrations/0018_auto_20200603_0620.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.0.6 on 2020-06-03 06:20
```

Comment:
in your commit message:
When a video is encoded, the "lambda-complete" lambda updates the upload
state and also sets the resolutions available for this video on the model.